### PR TITLE
Feat/test runner

### PR DESCRIPTION
**Pull request by mariush2**
Coverage will not be 100% until vitest guys fix the coverage merging when having more than 1 project: 
https://github.com/vitest-dev/vitest/issues/5783

[AB#534590](https://dev.azure.com/Equinor/1f7078da-0ba3-4461-a220-c3e39c5c1f09/_workitems/edit/534590)

Also created a migration guide here for browser mode: https://dev.azure.com/Equinor/Johan%20Sverdrup%20Digitalisation/_wiki/wikis/Johan-Sverdrup-Digitalisation.wiki/9779/Frontend-Vitest-browser-mode-in-app

### CODE REVIEW

File: src/organisms/SideBar/CreateItem.jsdom.test.tsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+      expect(button).toHaveStyleRule('height', '36px');
+      expect(button).toHaveStyleRule(
+        'border-radius',
+        shape.icon_button.borderRadius
+      );
+      expect(button).toHaveStyleRule(
+        'border-radius',
+        shape.icon_button.borderRadius,
+        { modifier: ':hover' }
+      );
```
Comment (aslakihle) [inline src/organisms/SideBar/CreateItem.jsdom.test.tsx:56]: I know you did nott write this test in this PR.
But this feels like an uncessesary amount of style checks for the test. I think it should only test for the things that change between collapsed/open etc 🤔

File: src/organisms/SideBar/ToggleOpen.jsdom.test.tsx [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+    expect(iconContainer).toHaveStyleRule('width', '32px');
+    expect(iconContainer).toHaveStyleRule('height: 32px');
+    expect(svgPath).toHaveAttribute('d', first_page.svgPathData);
+    expect(icon).toHaveAttribute('height', '24px');
+    expect(icon).toHaveAttribute('width', '24px');
+
+    expect(text).toHaveStyle('font-weight: 500');
+    expect(text).toHaveStyle(
+      `color: ${colors.text.static_icons__default.rgba}`
+    );
```
Comment (aslakihle) [inline src/organisms/SideBar/ToggleOpen.jsdom.test.tsx:52]: I guess the previous comment would apply here aswell

File: src/organisms/TopBar/TopBar.jsdom.test.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  );
+  expect(topBar).toHaveStyleRule('padding-left', spacings.medium);
+  expect(topBar).toHaveStyleRule('align-items', 'center');
+  expect(topBar).toHaveStyleRule('height', '64px');
+
+  expect(headerContainer).toHaveStyleRule('gap', spacings.medium);
+
+  expect(appIdentifier).toHaveStyleRule('display', 'flex');
+  expect(appIdentifier).toHaveStyleRule('align-items', 'center');
+  expect(appIdentifier).toHaveStyleRule('gap', spacings.medium_small);
```
Comment (aslakihle) [inline src/organisms/TopBar/TopBar.jsdom.test.tsx:56]: Again here many style tests, atleast they are only done once here, but still feels a little overkill 🤔

**Review by aslakihle — APPROVED** [review]
Approving even though i had some comments. 
These comments could be looked into later if we want to, but just wanted to point it out when i saw it c:

**Comment by mariush2** [comment]
@aslakihle Created a user story for the comments you had 🎉 https://dev.azure.com/Equinor/DPP%20Upscaling%20of%20digital%20solutions/_workitems/edit/579362

**Pull request merged**